apps/examples/fs_performance/fs_performance_main.c

```diff
@@ -0,0 +1,502 @@
+			if (ret != (f_sz)) {
+				printf("Unable to write in file: %s, size = %d, ret = %d\n", file[0], f_sz, ret);
+				close(fd);
+				return ERROR;
+			}
+                	close(fd);
+
+			frt_fd = open("/dev/timer0", O_RDONLY);
+			if (frt_fd < 0) {
+                                printf("Unable to do open file : /dev/timer0, errno : %d\n", file[0], errno);
+                                return ERROR;
+                        }
+			ioctl(frt_fd, TCIOC_SETMODE, MODE_FREERUN);
+			ioctl(frt_fd, TCIOC_START, TRUE);
+			ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)&before);
+
+			fd = open(file[0], O_WROK | O_APPEND);
+			if (fd < 0) {
+				printf("Unable to open file : %s, ret = %d\n", file[0], fd);
+				close(frt_fd);
+				return ERROR;
+			}
+
+			/*append 64 kb data*/
+			for (j = 0; j < (64 * 1024) / b_sz; j++ ) {
+				ret = write(fd, rw_buf, b_sz);
+				if (ret != (b_sz)) {
+					printf("Unable to append to file: %s, ret = %d\n", file[0], ret);
+					close(frt_fd);
+					close(fd);
+					return ERROR;
+				}
+			}
+			close(fd);
+
+			ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)&after);
+			ioctl(frt_fd, TCIOC_STOP, 0);
+			close(frt_fd);
+
+			tt2 = after.timeleft - before.timeleft;
+			tt += tt2;
+			printf("iteration : %d,time : %lu\n", it, tt2);
+
+			ret = unlink(file[0]);
+			if (ret != OK) {
+				printf("Failed to unlink file %s, ret : %d, errno : %d\n", file[0], ret, errno);
+				return ERROR;
+			}
+		}
+		printf("total time : %lu us\n", tt);
+		break;
+
+	case TEST_FLAG_READ :
+		memset(rw_buf, 0, BUFFER_SIZE);
+		fd = open(file[0], O_WROK | O_CREAT);
+                if (fd < 0) {
+			printf("Unable to open file : %s,  ret : %d\n", file[0], fd);
+                        return ERROR;
+                }
+
+                ret = write(fd, rw_buf, 512);
+                if (ret != (512)) {
+			printf("Unable to write in file: %s, ret : %d\n", file[0], ret);
+			close(fd);
+			return ERROR;
+                }
+		close(fd);
+		for (int file_size = 512; file_size <= (f_sz); file_size *= 2) {
+			fd = open(file[0], O_WROK | O_APPEND);
+			if (fd < 0) {
+				printf("Unable to do open file : %s,  ret = %d\n", file[0], fd);
+				return ERROR;
+                        }
+
+			for (int wr_i = 0; wr_i < (file_size / 1024); wr_i++ ) {
+				ret = write(fd, rw_buf, 512);
+				if (ret != (512)) {
+					printf("Unable to create  file: %s,size = %d ,ret = %d\n", file[0], file_size, ret);
+					close(fd);
+					return ERROR;
+                                 }
+                         }
+
+			for (int buf_size = 512; buf_size <= (b_sz); buf_size *= 2) {
+				tt = 0;
+
+				if (file_size < buf_size)
+					continue;
+
+				frt_fd = open("/dev/timer0", O_RDONLY);
+				if (frt_fd < 0) {
+					printf("Unable to do open file : /dev/timer0, errno : %d\n", file[0], errno);
+					return ERROR;
+				}
+
+				ioctl(frt_fd, TCIOC_SETMODE, MODE_FREERUN);
+                                ioctl(frt_fd, TCIOC_START, TRUE);
```

Comment:
Could you correct the indentation please?
Tab spaces everywhere instead of spaces